src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIncentiveReviewService.kt

```diff
@@ -219,8 +220,8 @@ class PrisonerIncentiveReviewService(
     }
   }
 
-  private suspend fun getIncentiveLevelForReviewType(prisonerInfo: PrisonerInfo, reviewType: ReviewType): String {
-    val prisonIncentiveLevels = prisonIncentiveLevelService.getActivePrisonIncentiveLevels(prisonerInfo.agencyId)
+  private suspend fun getIncentiveLevelForReviewType(prisonerInfo: PrisonerBasicInfo, reviewType: ReviewType): String {
```

Comment:
Technically I didn't _have to_ change the signature of this method as it's always invoked with the response from Prisoner Search API now (as the SNS domain event has the prisoner number)...

...but the method body only use basic information so it made sense to restrict the input to the things the method uses (it only uses `prisonId` and `bookingId` properties)